.github/workflows/bump-version.yml

```diff
@@ -39,8 +40,11 @@ jobs:
 
       - name: Bump Cloudbeat
         # bump_cloudbeat.sh will create multiple PRs with different HEAD branches
+        env:
+          GIT_BASE_BRANCH: ${{ github.base_ref || github.ref_name }}
         run: scripts/bump_cloudbeat.sh
 
       - name: Bump Cloud Security Posture Integration
         # we need to run bump_integration.sh from the main branch
+        if: github.base_ref == 'main'
```

Comment:
This condition enforces that the workflow must run on the `main` branch for the bump integration. If this is the case, we need to document this behavior and also update the documentation to describe the version bump process.